### DISCUSSION TITLE
opening a file manually is not saved to session

### DISCUSSION THREAD

**Discussion by semi710** [olimorris/persisted.nvim#109] [Q&A] [answered]
if i do `nvim` then open a file from there the output of `vim.g.persisting` is `true`
but if i do `nvim /path/to/file` then output of `vim.g.persisting` is nil.
now the /path/to/file is not saved in session.
how can i make it to get saved in session?

**Comment by olimorris** [comment, marked as answer]
Please see the README for extending the plugin to suit your specific needs
